packages/core/src/codewhisperer/views/lineAnnotationController.ts

```diff
@@ -262,8 +262,7 @@ export class LineAnnotationController implements vscode.Disposable {
                     try {
                         telemetry.ui_click.emit({ elementId: `dismiss_${this._currentState.id}` })
                     } catch (_) {}
-                    this._currentState = new EndState()
-                    await vscode.commands.executeCommand('setContext', inlinehintWipKey, false)
+                    await this.markTutorialDone()
```

Comment:
for reference 
https://github.com/aws/aws-toolkit-vscode/blob/master/packages/core/src/codewhisperer/views/lineAnnotationController.ts#L288

Reply:
```
    async markTutorialDone() {
        this._currentState = new EndState()
        await vscode.commands.executeCommand('setContext', inlinehintWipKey, false)
        await set(inlinehintKey, this._currentState.id, globals.context.globalState)
    }
```